clippy_utils/src/lib.rs

```diff
@@ -2234,7 +2234,7 @@ pub fn is_slice_of_primitives(cx: &LateContext<'_>, expr: &Expr<'_>) -> Option<S
 }
 
 /// returns list of all pairs (a, b) from `exprs` such that `eq(a, b)`
-/// `hash` must be comformed with `eq`
+/// `hash` must be verified with `eq`
```

Comment:
The original word was probably `confirmed`, but `verified` is better (someone else suggested it to me and I agree)